app/scripts/modules/core/src/pipeline/config/stages/bake/ManualExecutionBake.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import { HelpField } from 'core/help/HelpField';
+import { ITriggerTemplateComponentProps } from 'core/pipeline/manualExecution/TriggerTemplate';
+
+export class ManualExecutionBake extends React.Component<ITriggerTemplateComponentProps> {
+  private handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const target = event.target;
+    const checked = target.checked;
+    this.props.command.trigger.rebake = checked;
+    this.setState({});
+  };
+
+  public render() {
+    const force = Boolean(this.props.command.trigger.rebake) ? 'true' : 'false';
```

Comment:
why not just `const force = this.props.command.trigger.rebake === 'true';`

Reply:
Because we're doing the inverse, ya dingus.

Reply:
are we tho?